### PR TITLE
Navigation Breadcrumb For Officer Page

### LINKED ISSUE #582: Add shortcut to officer profile page to return to department
If someone is viewing the officer profile page and they want to return to the department page, there is currently no button to take them to the officer's department.  This seems like another issue that overlaps with #473.  Thoughts are welcome!  (I am not a design specialist.)

### PR DESCRIPTION
**Pull request by fritzdavenport**
## Status

Ready for review / in progress

## Description of Changes

Fixes #582 

Changes proposed in this pull request:

 - Navigation Breadcrumbs for Office Page

## Notes for Deployment

Nuns.

## Screenshots (if appropriate)

breadcrumbs

## Tests and linting

 - [x] I have rebased my changes on current `develop`

 - [x] pytests pass in the development environment on my local machine

 - [ ] `flake8` checks pass - broken on master???

### CODE REVIEW

File: OpenOversight/app/templates/officer.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <div class="container" role="main">
 
   <div class="page-header">
-    <h1>Officer Detail: <b>{% include 'partials/officer_name.html' %}</b></h1>
+      <h1><a href={{url_for('main.list_officer',department_id=officer.department.id)}}>{{ officer.department.name|title }}</a> >> <b>{% include 'partials/officer_name.html' %}</b></h1>
```
Comment (redshiftzero) [inline OpenOversight/app/templates/officer.html]: Looks good! So one idea: there are styled breadcrumbs in bootstrap (which we use) - see [here](https://getbootstrap.com/docs/3.3/components/#breadcrumbs), want to try using those? They look pretty nice

**Comment by fritzdavenport** [comment]
:+1: @redshiftzero, updated. Also added a `name` field 
breadcrumbs

**Comment by redshiftzero** [comment]
One last request: Can we preserve the title in the `<h1>` tag and put the breadcrumb below it? That _should_ also resolve the test failures

**Comment by fritzdavenport** [comment]
@redshiftzero  fixed
screenshot from 2018-12-17 01-59-52

**Pull request merged**